sling/nlp/kb/phrase-table-builder.cc

```diff
@@ -284,6 +284,7 @@ class PhraseTableBuilder : public task::FrameProcessor {
         const EntityPhrase &e = phrase->entities[i];
         const Entity &entity = entity_table_[e.index];
         Handle item = commons_->Lookup(entity.id);
+        CHECK(!item.IsNil()) << entity.id;
```

Comment:
Potentially also add 
```
        // Skip items that are not in the knowledge base.
        if (entity_item[source].IsNil()) continue;

```
inside the for-loop below.

Reply:
We are trying to prevent that entity_item[source] is never nil, but we check this in the loop above, so this should never happen.